src/config/index.js

```diff
@@ -31,7 +31,9 @@ export const functions = firebase.functions();
 let firebase_messaging;
```

Comment:
Haven't we gitignore this file?

Reply:
same issue happened here , this branch was created after merging into main branch ( after config file was gitignored). But still It's appearing in PRs.

Reply:
issue solved in #141